### PR TITLE
added initial documents for the lib skeleton

### PR DESCRIPTION
**Pull request by mike-ionut-mihai-sandu-tao**
initial files for creating a lib skeleton:

- .Jenkinsfile
- .gitignore
- CHANGELOG.md
- README.md
- composer.json
- phpunit.xml.dist
-

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,5 @@
+/vendor/
```
Comment (bartlomiejmarszal) [inline .gitignore]: Why do you need that?

File: README.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
+# lib-proctorio
+
+![TAO Logo](https://github.com/oat-sa/taohub-developer-guide/raw/master/resources/tao-logo.png)
+
+
+
+> Short description of `oat-sa/lib-proctorio`
```
Comment (bartlomiejmarszal) [inline README.md]: I think that we should replace this with short description

File: .gitignore (hunk 1 above)
Comment (mike-ionut-mihai-sandu-tao) [reply .gitignore]: "phpunit/phpunit": "^8.5"
also at some point we were requiring the ramsey:uuid lib

File: README.md (hunk 2 above)
Comment (mike-ionut-mihai-sandu-tao) [reply README.md]: can you be more specific ?

Comment (mike-ionut-mihai-sandu-tao) [reply README.md]: added small and long description

Comment (bartlomiejmarszal) [reply README.md]: I meant: 
```suggestion
```

File: .Jenkinsfile [hunk 3]
```diff
@@ -0,0 +1,33 @@
+pipeline {
+    agent {
+        label 'master'
```
Comment (bartlomiejmarszal) [inline .Jenkinsfile]:
```suggestion
        label 'builder'
```

Comment (mike-ionut-mihai-sandu-tao) [reply .Jenkinsfile]: why ?

File: .Jenkinsfile [hunk 4]
```diff
@@ -0,0 +1,33 @@
+pipeline {
+    agent {
+        label 'builder'
```
Comment (mike-ionut-mihai-sandu-tao) [inline .Jenkinsfile]: According to @serge-salamanka-1pt this should never be master but always builder
because master is forbidden

**Pull request merged**